Resources/Prototypes/_DV/Entities/Structures/Wallmounts/flags.yml

```diff
@@ -0,0 +1,116 @@
+- type: entity
+  parent: BaseFlag
+  id: AceFlag
+  name: asexual flag
+  description: '"Be ace do crime" flag.'
+  components:
+  - type: Sprite
+    sprite: _DV/Structures/Wallmounts/flags.rsi
```

Comment:
put the sprite path in a abstract BaseFlagDV and change these to inherit from it instead of copy pasting it for them all